packages/geoprocessing/scripts/dataPrep/bundleFeatures.ts

```diff
@@ -121,7 +124,7 @@ const bundleFeatures = async (
     let processedCount = 0;
     for (const feature of records) {
       sumNPoints += feature.npoints;
-      extent = expandBBox(extent, feature.bbox);
+      extent = expandBBox(extent, feature.bbox as BBox);
```

Comment:
I didn't find a good way to get the zod type to match the BBox type so I punted and did a couple casts.